src/main/java/seedu/commands/BudgetCommand.java

```diff
@@ -96,5 +96,44 @@ private void addBudget(SpendingList spendings, int budget, String timeRange) {
             throw new WiagiInvalidInputException(INVALID_CATEGORY + BUDGET_COMMAND_FORMAT);
```

Comment:
Is it `INVALID_CATEGORY`? I think its `INVALID_FIELD` as we used `CATEGORY` to explain `spendings` and `incomes` for most part of the code, while field is for the possible options of the command